index.html

```diff
@@ -175,6 +175,21 @@ <h5>Skills</h5>
         </ul>
         </section>
 
+        <section class="contact_form">
```

Comment:
As per the Figma template provided to you, there is a certain gap between the `Skills` section and `Contact` section. However, in your design, there is no such gap. Kindly add some margin between the two. 

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://github.com/medijdir/mobileportfolio/assets/33268927/1ccfcab8-851d-4b27-ad2b-44d03b9685f1) | ![image](https://github.com/medijdir/mobileportfolio/assets/33268927/f323d442-2c87-4ec4-afa7-01c7214dfd0e) |